src/comments/comment.entity.ts

```diff
@@ -0,0 +1,74 @@
+import {
+  Column,
+  CreateDateColumn,
+  DeleteDateColumn,
+  Entity,
+  JoinColumn,
+  ManyToOne,
+  PrimaryGeneratedColumn,
+  UpdateDateColumn,
+} from 'typeorm';
+import { User } from '../users/users.legacy.entity';
+
+@Entity()
+export class Comment {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column()
+  content: string;
+
+  @Column()
+  commentableId: number;
+
+  @Column()
+  commentableType: 'answer' | 'comment' | 'question';
+
+  @Column()
+  agreeCount: number;
+
+  @Column()
+  disagreeCount: number;
+
+  @ManyToOne(() => User)
+  @JoinColumn({ name: 'userId' })
+  user: User;
+
+  @Column()
+  userId: number;
+
+  @CreateDateColumn()
+  createdAt: Date;
+
+  @UpdateDateColumn()
+  updatedAt: Date;
+
+  @DeleteDateColumn()
+  deletedAt?: Date;
+}
+
+@Entity()
+export class UserAttitudeOnComments {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @ManyToOne(() => User)
+  @JoinColumn({ name: 'userId' })
+  user: User;
+
+  @Column()
+  userId: number;
+
+  @ManyToOne(() => Comment)
+  @JoinColumn({ name: 'commentId' })
+  comment: Comment;
+
+  @Column()
+  agreeType: '3' | '1' | '2';
```

Comment:
If the user has no attitude towards a comment, then the record should be deleted, rather than setting to '3'